xbmc/music/MusicDatabase.cpp

```diff
@@ -42,7 +42,6 @@
 #include "messaging/helpers/DialogOKHelper.h"
 #include "music/tags/MusicInfoTag.h"
 #include "network/Network.h"
-#include "network/cddb.h"
```

Comment:
This header, to be guarded by "#ifdef HAS_DVD_DRIVE", is still needed and will fix the jenkins build error.

Reply:
Thanks! For some reason it works without it on my systems. Maybe full rebuild wasn't triggered.